pkg/api/http_server.go

```diff
@@ -433,6 +433,11 @@ func (hs *HTTPServer) addMiddlewaresAndStaticRoutes() {
 	m.Use(middleware.Recovery(hs.Cfg))
 
 	hs.mapStatic(m, hs.Cfg.StaticRootPath, "build", "public/build")
+
+	if hs.Features.IsEnabled(featuremgmt.FlagSwaggerUi) {
+		hs.mapStatic(m, hs.Cfg.HomePath, "swagger-ui", "swagger-ui")
```

Comment:
Not sure if using `hs.Cfg.HomePath` is correct since it's the data folder supposed to store configuration data and can be configured/is not stable, https://github.com/grafana/grafana/blob/12420260ef35068c7244313d4a840f62848a84e1/pkg/setting/setting.go#L789.

Reply:
I'm not sure about this, therefore I decided to make it configurable so that users are able to change it.

Reply:
I'm not sure that makes sense. Guess it comes down to how you're planning to package this when releasing a new version of Grafana. 

With the feature toggle is the point to refuse serve the swagger-ui solely or both swagger-ui and .json files? To me would be enough with swagger-ui. What if we put the .json files under public and allow them to be served even if feature toggle is disabled - you can already download it from the repository, right. 

Then, add swagger-ui index.html to the [views directory](https://github.com/grafana/grafana/tree/main/public/views). Register a route similar to index page and only render it if feature toggle is enabled.
https://github.com/grafana/grafana/blob/875e0736ec990be6369de8e643bfcf4287a6e48a/pkg/api/index.go#L663-L670

Thoughts?

Reply:
thanks for this tip; I'll try it out!

Reply:
I think it worked 🎉  I've pushed the changes, thanks again for the help!